src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 5;
+    private static final int CIRCLE_RADIUS = 10;
+    private static final String CIRCLE_COLOR = "white";
+    private final ColorSupplier colorSupplier;
+    private Random random;
+
+    public FigureSupplier(ColorSupplier colorSupplier, Random random) {
+        this.colorSupplier = colorSupplier;
+        this.random = random;
+    }
+
+    private int getRandomValues() {

```

Comment:
```suggestion
    private int getRandomValue() {
```
better to use singular form as this method return only one value